city_scrapers/spiders/chi_ssa_22.py

```diff
@@ -0,0 +1,124 @@
+# import logging
+import unicodedata
+from datetime import datetime
+
+import dateutil.parser
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa22Spider(CityScrapersSpider):
+    name = "chi_ssa_22"
+    agency = "Chicago Special Service Area #22 Clark St/Andersonville"
+    timezone = "America/Chicago"
+    start_urls = ["http://www.andersonville.org/our-organizations/andersonville-ssa-22/"]
+    location = {
+        "name": "Andersonville Chamber of Commerce",
+        "address": "5153 N. Clark St. #228 Chicago, Illinois 60640"
+    }
+    time = '9:30am'
+
+    def parse(self, response):
+        # the address is in a normal paragraph, so whole page is looked at
+        self._validate_location(response.body.decode('utf-8'))
+
+        h2s = response.xpath('//h2')
+
+        general_desc = 'All meetings are held at the Andersonville ' \
+                       'Chamber of Commerce conference room'
+
+        # Dictionary containing all meeting dictionaries
+        # The dates will be the keys
+        meetings = dict()
+
+        last_year = datetime.today().replace(year=datetime.today().year - 1)
+
+        for entry_cnt, entry in enumerate(h2s, start=1):
+            entry_str = entry.xpath('./text()').extract_first()
+            if entry_str and ('Meeting Schedule' in entry_str or 'Meetings & Minutes' in entry_str):
+                year = entry_str[0:4]
+                # logging.debug(year)
```

Comment:
This and the other commented out logging calls can be removed

Reply:
True. I have removed them.